pkg/batcher/handler.go

```diff
@@ -173,7 +174,7 @@ func (handler *BatchHandler) batch() {
 				index,
 			}
 			handler.batcherInfo.CurrentInputLen = len(handler.batcherInfo.Instances)
-		case <-time.After(SleepTime):
+		case <-time.After(time.Duration(handler.Timeout)):
```

Comment:
This is not actually a timeout for batching, it is a tick time for the select loop. `maxLatency` is the batcher timeout.

Reply:
I had this impression but wasn't sure what the original intention was. Should I remove `BatcherTimeoutInternalAnnotationKey` annotation and the use of `BatcherArgumentTimeout` flag since they are not used and redundant to the existing `maxLatency` flag?

Reply:
I have updated the PR to remove these instead. Let me know if I have missed anything here.